README.md

```diff
@@ -1,2 +1,57 @@
-# swift_ingest
+# SWIFT ingest Ruby gem
 Ruby gem for storing files in swift openstack storage
+
+![Ruby Swift Ingest gem Diagram](docs/images/overview.png)
+
+## Requirements
+
+Swift_Ingest supports Ruby 2.3.1+
+
+## Installation
+
+Swift_Ingest is hosted on rubygems.org. Therefore it can be installed via:
+
+```bash
+  gem install swift_ingest
+```
+
+## Usage
+
+This gem in indented to be used from other ruby programs.
+Typical usage involves creating new SwiftIngest object
+
+```bash
+  swift_depositer = SwiftIngest::Ingestor.new(username: 'user',
```

Comment:
The variables are set in the lib/swift_ingest.rb file? Do we need a simplified `new` method so that when use the gem, it doesn't need to repeat all the variables again in the method?

Reply:
DEFAULTS are set to be used with the local version of swift docker running. It is not useful for real use. You need "new" method to pass all connection parameters.